src/test/scala/uk/gov/hmrc/test/ui/cucumber/runner/RunnerForDeparture.scala

```diff
@@ -24,6 +24,6 @@ import org.junit.runner.RunWith
   features = Array("src/test/resources/features"),
   glue = Array("uk.gov.hmrc.test.ui.cucumber.stepdefs"),
   plugin = Array("pretty", "html:target/cucumber", "json:target/cucumber.json"),
-  tags = "@departureGBe2e or @departureXIe2e or @departureAmendment"
+  tags = "@departureGBe2e or @departureXIe2e"
```

Comment:
Wasn't this just changed to add the departureAmendment tag? Not sure why its being removed?